app/accounts/tests/graphql/test_visitor.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+from graphql_jwt.testcases import JSONWebTokenTestCase
+from django.test import TestCase
+from graphene.test import Client
+from alohomora.schema import schema
+from django.contrib.auth import get_user_model
+from accounts.models import Visitor
```

Comment:
Não esqueça as [docstings de módulo](https://stackoverflow.com/questions/2557110/what-to-put-in-a-python-module-docstring#2557196)